models/Nous-Hermes-Llama2-13b.json

```diff
@@ -40,12 +40,12 @@
           "repositoryUrl": "https://huggingface.co/TheBloke/Nous-Hermes-Llama2-GGML"
         },
         {
-          "name": "nous-hermes-llama2-13b.ggmlv3.q8_0.bin",
-          "url": "https://huggingface.co/TheBloke/Nous-Hermes-Llama2-GGML/resolve/main/nous-hermes-llama2-13b.ggmlv3.q8_0.bin",
-          "sizeBytes": 13831378592,
-          "quantization": "q8_0",
+          "name": "nous-hermes-llama2-13b.ggmlv3.q6_K.bin",
+          "url": "https://huggingface.co/TheBloke/Nous-Hermes-Llama2-GGML/resolve/main/nous-hermes-llama2-13b.ggmlv3.q6_K.bin",
+          "sizeBytes": 10830311072,
+          "quantization": "q6_K",
```

Comment:
I believe the json schema expects `Q6_K` (upper case Q)

Reply:
Ah, I forgot to run the validation script before pushing!

Reply:
Btw on my machine I need to prefix it like so:
```bash
 PYTHON=/usr/bin/python3 python validate.py
```
but you might not need to

Reply:
Thanks! Worked without prefixing it.
Two files were failing the check. I've fixed them and pushed them again!